lib/nunjucks/govuk-prototype-kit/layouts/govuk-branded.njk

```diff
@@ -18,7 +18,8 @@
     homepageUrl: "/",
     serviceName: serviceName,
     serviceUrl: "/",
-    containerClasses: "govuk-width-container"
+    containerClasses: "govuk-width-container",
+    rebrand: govukRebrand
```

Comment:
Again I think this affects both the 'manage prototype' area and the prototypes themselves 😕

Reply:
Could we modify the `nunjucksManagementEnv` in `lib/manage-prototype-handlers.js` to set the `govukRebrand` global instead? 🤔